packages/types/src/index.ts

```diff
@@ -279,6 +290,18 @@ export namespace Resolver {
   }
 }
 
+export namespace Transfer {
```

Comment:
keep only Target, Targets, Entry, as part of Transfer, move everything else to namespace Chain, namespace Cardano -> Chain.Cardano.{Address,UnsignedTx,TxInfo,Metadata,Staking,Voting,Address}